numpy/core/setup.py

```diff
@@ -674,6 +674,13 @@ def get_mathlib_info(*args):
     # Intel and Clang also don't seem happy with /GL
     is_msvc = (platform.platform().startswith('Windows') and
                platform.python_compiler().startswith('MS'))
+    # issue 16744: remove this code  and the call to call_emms in nypmath
+    # when there is a fix from microsoft
+    if is_msvc:
+        npymath_sources.append(join('src', 'npymath', 'call_emms.obj'))
+    elif platform.platform().startswith('Windows'):
+        npymath_sources.append(join('src', 'npymath', 'call_emms.asm'))
+
```

Comment:
is `asm` the correct suffix to get mingw to compile assembly code? Also: I added a `call_emms_d.obj` for debug builds but it is not used here yet